src/frameworks/drupal7/composer-manager.md

```diff
@@ -22,23 +22,19 @@ Composer Manager works by using a Drush command to aggregate all module-provided
 
 ```php
 $conf['composer_manager_vendor_dir'] = 'composer/vendor';
-$conf['composer_manager_file_dir'] = 'composer/';
+$conf['composer_manager_file_dir'] = 'composer';
 ```
 
 The above lines will direct Composer Manager to put the generated `composer.json` file into `composer/`, and to autoload Composer-based packages from the `composer/vendor` directory.  The paths are relative to the Drupal root.  You may use another location if desired (such as `../composer`), provided that they are not in the file mount and the `vendor` directory is a sibling of wherever the `composer.json` file will be.
 
-Then, manually create the `composer` and `composer/vendor` directories and place an empty file named `.gitkeep` into the vendor directory, then commit it to git.  Composer Manager needs that directory to exist but due to a bug will not create it itself.  The file name technically does not matter but `.gitkeep` is a very common convention.
-
-## 3.  Configure `.gitignore`
-
-If you have a `.gitignore` file, The vendor directory should be added to it.  However, due to the `.gitkeep` file it is a bit trickier than just listing the directory name.  Adding the following two lines (adjusting for the path to the `composer.json` file as needed) will have the desired result, once `.gitkeep` is added to the repository:
+Then, manually create the `composer` directory and place a `.gitignore` file inside it, containing the following, and commit it to Git:
 
 ```
-composer/vendor/*/
-composer/vendor/autoload.php
+# Exclude Composer dependencies.
+/vendor
```

Comment:
This approach won't create a vendor directory, which in my testing needs to pre-exist.  Composer Manager won't create it. (This seems like a bug in Composer Manager, but I didn't dig too deeply into it.)